configure.ac

```diff
@@ -0,0 +1,383 @@
+m4_define([LIBARCHIVE_VERSION_S],[3.7.2])
```

Comment:
This file is a partial rip-off of the upstream configure.ac... so it might diverge in the future (just like config.h).